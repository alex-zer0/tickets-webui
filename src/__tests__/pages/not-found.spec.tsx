import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../../pages/NotFound';

describe('PAGES: NotFound', () => {
  it('renders w/o exceptions', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
