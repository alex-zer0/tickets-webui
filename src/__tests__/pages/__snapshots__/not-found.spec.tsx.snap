// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PAGES: NotFound renders w/o exceptions 1`] = `
<h1>
  404 Not Found
</h1>
`;
